<?xml version="1.0"?>

<launch>
	
  <node name="read_label" pkg="ftfootb_label_reading" ns="/label_reading" type="read_label" output="screen"> <!--launch-prefix="valgrind"-->
    <rosparam command="load" file="$(find ftfootb_label_reading)/ros/launch/read_label_params.yaml"/>
    <remap from="~colorimage_in" to="/camera/image_raw"/>
  </node>

</launch>